backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_Invoice_Details_local_currency_conversion.sql

```diff
@@ -0,0 +1,119 @@
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Sales_Invoice_Details_local_currency_conversion(IN p_C_invoice_id numeric)
+    RETURNS TABLE
+            (
+                c_invoice_id                numeric(10),
+                IsPrintLocalTaxReporting    boolean,
+                local_iso_code              bpchar(3),
+                conversion_rate             numeric,
+                conversion_date             timestamp without time zone,
+                local_taxbaseamt            numeric,
+                local_taxamt                numeric
+            )
+AS
+$$
+
+
+SELECT it.C_Invoice_ID,
+       CASE
+           WHEN i.m_inout_id IS NULL OR i.c_order_id IS NULL   THEN FALSE
+           WHEN i.isprintlocalcurrencyinfo = 'N'               THEN FALSE
+           WHEN bp.isprintlocalcurrencyinfo = 'N'
+               AND   i.isprintlocalcurrencyinfo <> 'Y'        THEN FALSE
+                                                               ELSE TRUE
+       END             AS IsPrintLocalTaxReporting,
+       cu.iso_code AS local_iso_code,
+       currencyrate(c.c_currency_id,
+                    cu.c_currency_id,
+                    getlocaltaxreportingconversionratedate(tr.taxreportingratebase, tr.c_calendar_id, i.dateinvoiced, m.movementdate),
+                    tr.c_conversiontype_id,
+                    i.ad_client_id,
+                    i.ad_org_id) AS conversion_rate,
+       CASE
+           WHEN i.m_inout_id IS NULL THEN NULL
+                                     ELSE
+                                         getlocaltaxreportingconversionratedate(
+                                                 tr.taxreportingratebase,
+                                                 tr.c_calendar_id,
+                                                 i.dateinvoiced,
+                                                 m.movementdate)
+       END AS conversion_date,
+       CASE
+           WHEN i.m_inout_id IS NULL THEN NULL
+                                     ELSE
+                                         currencyconvert(SUM(TaxBaseAmt),
+                                                         c.c_currency_id,
+                                                         cu.c_currency_id,
+                                                         i.dateinvoiced,
```

Comment:
why aren't we use conversion_date here?

Reply:
forget to change it back after testing something